packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -919,7 +919,7 @@ class RenderParagraph extends RenderBox
           } else if (recognizer is LongPressGestureRecognizer) {
             configuration.onLongPress = recognizer.onLongPress;
           } else {
-            assert(false);
+            assert(false, '${recognizer.runtimeType} is not supported.');
```

Comment:
Maybe: must be one of Tap/Longpress?

Reply:
Assets can be caught in unit tests too, just saying :)

Reply:
I didn't want to be too specific here, as I don't see why we cant support DoubleTap or MultiTap as well. If we add it shortly, then that would be overly specific and not strictly correct